test/util_api.go

```diff
@@ -150,15 +141,6 @@ func PauseContainerOk(c *check.C, cname string) {
 	CheckRespStatus(c, resp, 204)
 }
 
-// UnpauseContainerOk unpauses the container and asserts success..
```

Comment:
For this part, I do not think removal of code is a good idea. Instead, I am afraid that we should try to take advantages of that in the test code, like:
```
	// unpause it
	resp, err = request.Post("/containers/" + cname + "/unpause")
	c.Assert(err, check.IsNil)
	CheckRespStatus(c, resp, 204)
```

Reply:
In addition, maybe in another pr, we could use `PauseContainerOk` to replace 
```
	resp, err := request.Post("/containers/" + cname + "/pause")
	c.Assert(err, check.IsNil)
	CheckRespStatus(c, resp, 204)
```

Reply:
good catch, replace it in the next patch.